packages/riipen-ui/src/components/TableBody.test.jsx

```diff
@@ -0,0 +1,80 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import TableBody from "./TableBody";
+import Table from "./Table";
+
+describe("<TableBody>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(
+        <Table>
+          <TableBody />
+        </Table>
+      );
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(
+      <Table>
+        <TableBody />
+      </Table>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const child = <td>Test</td>;
+      const childContent = <tr>{child}</tr>;
```

Comment:
I think you could just combine these into `<tr><td>Test</td></tr>` since we aren't actually referencing `child` anywhere.